android/app/src/main/kotlin/com/zulip/flutter/ZulipPlugin.kt

```diff
@@ -221,7 +221,7 @@ private class AndroidNotificationHost(val context: Context)
                 val style = NotificationCompat.MessagingStyle(toAndroidPerson(messagingStyle.user))
                     .setConversationTitle(messagingStyle.conversationTitle)
                     .setGroupConversation(messagingStyle.isGroupConversation)
-                messagingStyle.messages.forEach { it?.let {
+                messagingStyle.messages.forEach { it.let {
```

Comment:
After this change, the `it.let {` isn't doing anything, so that layer of wrapping can be cut out.

I'll do that in a separate commit, since I'm revising this to add text to the commit messages anyway.